core/src/main/scala/cats/data/NonEmptyList.scala

```diff
@@ -0,0 +1,68 @@
+package cats
+package data
+
+import cats.std.list._
+//it needs a Functor
+//it needs semigroup - combine, filter
+
+/**
+  *  A data type which represents a single element (head) and some other
+  * structure (tail).
+  */
+final case class NonEmptyList[A](head: A, tail: List[A]) {
+
+  /**
+    * Return the head and tail into a single list
+    */
+  def unwrap: List[A] = head :: tail
```

Comment:
Thanks for the comments @ceedubs.. I'm learning!
I think it makes sense to call it `toList`. I was just trying to follow `OneAnd.scala`, thought it was a convention